autopush/router/simple.py

```diff
@@ -137,9 +145,27 @@ def route_notification(self, notification, uaid_data):
             returnValue(RouterResponse(response_body="Delivered"))
         else:
             self.metrics.increment("router.broadcast.miss")
+            if self.udp is not None:
+                yield deferToThread(self._send_udp_wake,
+                                    self.udp)
             returnValue(RouterResponse(202, "Notification Stored"))
 
-    #############################################################
+    def _send_udp_wake(self, udp_info):
```

Comment:
It might make sense to move this logic into a `SimpleRouter` class, and expose it as a separate router type. On the other hand, that'd actually be more complex than just the one conditional you have here (and YAGNI). What do you think?